clang/tools/clang-format/git-clang-format

```diff
@@ -869,5 +875,12 @@ def convert_string(bytes_input):
         return str(bytes_input)
 
 
+def print_filename(filename, null=False):
```

Comment:
Why setting a value for null?

Reply:
I think that was a reflex, I set it to `False`, which is what `opts.null` is by default. Do you want me to change it to remove the default value?

Reply:
If this were C++ I'd say yes, you touched all calls (since you added it). Don't know if in Python it is done differently.

Reply:
I removed the default value of the argument in c47ce3ca8946f81f39757c82bfa5eb4cddc863a5.